File: supporting docs/UnColored .NET/src/uncolored/controllers/NewAnalysisController.cs
namespace uncolored.controllers
{
	using ActionEvent = javafx.@event.ActionEvent;
	using FXML = javafx.fxml.FXML;
	using FXMLLoader = javafx.fxml.FXMLLoader;
	using Initializable = javafx.fxml.Initializable;
	using VBox = javafx.scene.layout.VBox;

	public class NewAnalysisController : Initializable
	{

//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @FXML private javafx.scene.layout.VBox vbox;
		private VBox vbox;
		private VBox nextFxml;

		public override void initialize(URL url, ResourceBundle rb)
		{
			try
			{
				// TODO
				nextFxml = FXMLLoader.load(this.GetType().getResource("/uncolored/views/NewAnalysis_1.fxml"));
			}
			catch (IOException ex)
			{
//JAVA TO C# CONVERTER WARNING: The .NET Type.FullName property will not always yield results identical to the Java Class.getName method:
				Logger.getLogger(typeof(NewAnalysisController).FullName).log(Level.SEVERE, null, ex);
			}
		}

//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @FXML private void next(javafx.event.ActionEvent event)
		private void next(ActionEvent @event)
		{
			vbox.Children.clear();
			vbox.Children.All = nextFxml;
		}

	}

}

File: supporting docs/Uncolored (Risks Assesment Tool)/src/uncolored/controllers/NewAnalysisController.java
package uncolored.controllers;

import java.io.IOException;
import java.net.URL;
import java.util.ResourceBundle;
import java.util.logging.Level;
import java.util.logging.Logger;
import javafx.event.ActionEvent;
import javafx.fxml.FXML;
import javafx.fxml.FXMLLoader;
import javafx.fxml.Initializable;
import javafx.scene.layout.VBox;

public class NewAnalysisController implements Initializable {

    @FXML
    private VBox vbox;
    private VBox nextFxml;

    @Override
    public void initialize(URL url, ResourceBundle rb) {
        try {
            // TODO
            nextFxml = FXMLLoader.load(getClass().getResource("/uncolored/views/NewAnalysis_1.fxml"));
        } catch (IOException ex) {
            Logger.getLogger(NewAnalysisController.class.getName()).log(Level.SEVERE, null, ex);
        }
    }

    @FXML
    private void next(ActionEvent event) {
        vbox.getChildren().clear();
        vbox.getChildren().setAll(nextFxml);
    }

}
